### DISCUSSION TITLE
405 Method Not Allowed

### DISCUSSION THREAD

**Discussion by NicolasChusseau** [supabase/supabase#14364] [Questions]
Hello everyone, 
I have an app using redwood, supabase and prisma.

I've just deployed it on vercel and now I can't access to my data. I have an error 405 Method Not Allowed, but I don't understand why, as it works well on localhost.
I didn't forget to add env var.

Actual : 
![image](https://github.com/supabase/supabase/assets/122353423/d85f0d30-1643-46d1-a221-de1f88bb1de1)
![image](https://github.com/supabase/supabase/assets/122353423/d2f6cc0f-41ce-4ade-9588-ed488e0efc89)

Expected (it works on localhost) : 
![image](https://github.com/supabase/supabase/assets/122353423/da0737f2-b8f0-4947-bd48-c1f544e6e11f)

If you have any idea, please tell me.

**Comment by souvik666** [comment]
hey please follow this #14393

**Reply by NicolasChusseau** [reply]
This is not the same issue. Here the problem appear after the deployment on vercel, on this other one, the problem appears after a migration.
